frontend/src/metabase/dashboard/utils.unit.spec.js

```diff
@@ -32,6 +37,19 @@ describe("Dashboard utils", () => {
       const result = await fetchDataOrError(failedFetch);
       expect(result.error).toEqual(error);
     });
+
+    it("should return true if any database has actions enabled", () => {
+      const dbData = [
+        { id: 1, settings: { "database-enable-actions": true } },
+        { id: 2, settings: { "database-enable-actions": true } },
+        { id: 3, settings: { "database-enable-actions": false } },
+      ];
+
+      const databases = dbData.map(createMockDatabase);
+
+      const result = isDatabaseActionsEnabled(databases);
+      expect(result).toBe(true);
+    });
```

Comment:
Let's add a test for a negative scenario too

Reply:
Nice, added a database to each test as well where the settings key is also undefined, since we can run into that as well (if the database has nothing set for the `settings` key)